### ISSUE TITLE
Possible error in documentation, in defer.html

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2306]
|<img alt="roup's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| roup reported|
|-|-|
|Trac ID|trac#2306|
|Type|defect|
|Created|2006-12-19 22:30:32Z|

- Page: http://twistedmatrix.com/projects/core/documentation/howto/defer.html

- Section: Multiple callbacks

The example ends with::

  # this series of callbacks and errbacks will print "Result: 12"
  g = Getter()
  d = g.getData(4)
  d.addCallback(printData)
  d.addErrback(printError)

The line::

  d = g.getData(4)

should be::

  d = g.getDummyData(4)

<details><summary>Searchable metadata</summary>

```
trac-id__2306 2306
type__defect defect
reporter__roup roup
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__Documentation Documentation
time__1166567432000000 1166567432000000
changetime__1166904504000000 1166904504000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set status to `closed`|
|-|-|

(In [19047]) Apply fix for Deferred documentation bug

Author: roup
Reviewer: exarkun
Fixes [#2306](https://github.com/twisted/twisted/issues/2306)

Replace erroneous "getData" call with a call to the function actually defined
in the documentation, "getDummyData".

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
